app/src/main/java/de/xikolo/controllers/section/VideoPreviewFragment.kt

```diff
@@ -113,6 +131,24 @@ class VideoPreviewFragment : ViewModelFragment<VideoPreviewViewModel>() {
 
         textTitle.text = item.title
 
+        if (VideoDescriptionDelegate.isVideoSummaryAvailable(video.summary)) {
```

Comment:
That's not how delegates were meant to be used. They should only be used by ViewModels and calls be wrapped.